sheet2linkml/terminologies/tccm/api.py

```diff
@@ -40,7 +40,12 @@ def get_enum_values_for_field(self, field_name: str):
         response = requests.get(
             url, headers={"accept": "application/x-yaml"}, params={"value_only": "true"}
         )
-        if not response.ok:
+        if response.status_code == 404:
+            logging.debug(
+                f"Field not found on TCCM Terminology Service ({field_name}): {response}"
+            )
+            return {}
+        elif not response.ok:
```

Comment:
Also present in PR #21. So be sure to resolve merge conflicts accordingly when that PR is merged.

Reply:
Yeah, I think that's going to be something interesting to do together at tomorrow's pair programming session! I would add a test to make sure we catch this, but hopefully the test in the PR we're working on tomorrow will ensure that this works :)